src/relay/backend/build_module.cc

```diff
@@ -491,8 +491,8 @@ class RelayBuildModule : public runtime::ModuleNode {
     auto lowered_funcs = executor_codegen_->GetIRModule();
 
     // No need to build for external functions.
-    if (lowered_funcs.find("ext_dev") != lowered_funcs.end()) {
-      lowered_funcs.Set("ext_dev", IRModule());
```

Comment:
@Mousius @junrushao1994 Is this the right way to check if the target is ext_dev? I wasn't sure